tests/dapr_tests.mk

```diff
@@ -221,6 +221,12 @@ test-e2e-all: check-e2e-env test-deps
 	# Note2: use env variable DAPR_E2E_TEST to pick one e2e test to run.
 	DAPR_CONTAINER_LOG_PATH=$(DAPR_CONTAINER_LOG_PATH) GOOS=$(TARGET_OS_LOCAL) DAPR_TEST_NAMESPACE=$(DAPR_TEST_NAMESPACE) DAPR_TEST_TAG=$(DAPR_TEST_TAG) DAPR_TEST_REGISTRY=$(DAPR_TEST_REGISTRY) DAPR_TEST_MINIKUBE_IP=$(MINIKUBE_NODE_IP) gotestsum --jsonfile $(TEST_OUTPUT_FILE_PREFIX)_e2e.json --junitfile $(TEST_OUTPUT_FILE_PREFIX)_e2e.xml --format standard-quiet -- -p 2 -count=1 -v -tags=e2e ./tests/e2e/$(DAPR_E2E_TEST)/...
 
+# start selected e2e tests.
+test-e2e-selected: check-e2e-env test-deps
```

Comment:
This can be super useful used in the existing target. This allows us to change the tests to run and still use the same workflow file in GitHub. Also, it would be great to keep it compatible with the existing DAPR_E2E_TEST. I think this would mean adding "if" statements in the existing test e2e target.

Reply:
fixed.